ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -8,6 +8,8 @@ parameters:
   required: true
 - name: NAMESPACE
   required: true
+- name: IMAGE_FORMAT
+  required: true
```

Comment:
It should not be required - the template is used in contexts where it may be empty.  In the contexts of PR jobs it will be filled, but outside of PR jobs it will not be.

Reply:
Fixed